site/src/styles/_layout.scss

```diff
@@ -1,8 +1,23 @@
-@import "~@massds/mayflower-react/es/assets/scss/01-atoms/global/elements";
+@use 'bootstrap-grid';
 
-@import "~@massds/mayflower-react/es/assets/scss/04-templates/general";
+@use "globe" as *;
 
-@import "~@massds/mayflower-react/es/assets/scss/08-print/print";
+@use "00-base/layout";
+
+@use "00-base/fonts";
+@use "00-base/fonts-fallback";
+@use "00-base/fonts-styles";
+
+@use "00-base/elements";
+
+@use "04-templates/general";
+@use "08-print/print";
+@use "03-organisms/header-slim";
+@use "01-atoms/site-logo";
+@use "01-atoms/image";
+@use "03-organisms/footer-slim";
+@use "01-atoms/svg-icons";
+@use "01-atoms/svg-loc-icons";
 
 html {
```

Comment:
from here down to `// header search selectbox override` can be removed, same as `@use "00-base/elements";`